app/controllers/api/v2/hosts_controller.rb

```diff
@@ -336,7 +336,12 @@ def host_attributes(params, host = nil)
             interface_attributes(nic_attr, allow_nil_type: host.nil?)
           end
         end
-        params = host.apply_inherited_attributes(params) if host
+
+        if host
+          params = host.apply_inherited_attributes(params) if host
```

Comment:
is this the same `if host` as in the line above? :)

Reply:
oh I thought I already removed this, good catch, thanks :D

Reply:
Fixed!